roles/KubevirtTemplateValidator/tasks/main.yml

```diff
@@ -1,5 +1,18 @@
 ---
 # tasks file for KubevirtTemplateValidator
+- name: Extract the operator POD info
```

Comment:
Could you please extract this to a "playbook" and set_fact the operator_registry from it. That way all the other playbooks can use include_tasks to get this functionality and then easily use just the variable.

Reply:
no problem, I was just unsure about the right way to share code. Let's try.

Reply:
The other option (maybe even better) might be to make this into a role.

Reply:
went for making this into a role